src/controllers/controller.doughnut.js

```diff
@@ -272,7 +272,7 @@ export default class DoughnutController extends DatasetController {
       const options = this.resolveDataElementOptions(i);
       max = Math.max(max, options.offset || 0, options.hoverOffset || 0);
     }
-    return max;
+    return max * 1.5;
```

Comment:
How was `1.5` determined here?

Reply:
Each arc is offset from center 1/4 offset
https://github.com/chartjs/Chart.js/blob/7432b609c487e1b17c8000fe9f94c58fa65936b8/src/elements/element.arc.js#L324
https://github.com/chartjs/Chart.js/blob/7432b609c487e1b17c8000fe9f94c58fa65936b8/src/elements/element.arc.js#L337
https://github.com/chartjs/Chart.js/blob/7432b609c487e1b17c8000fe9f94c58fa65936b8/src/elements/element.arc.js#L339
Except when the arc measures more than 180 degrees, in that case is 1/2 offset
https://github.com/chartjs/Chart.js/blob/7432b609c487e1b17c8000fe9f94c58fa65936b8/src/elements/element.arc.js#L340-L342
In addition to the separation from the center to the radius of the arc, the radiusOffset is increased again
https://github.com/chartjs/Chart.js/blob/7432b609c487e1b17c8000fe9f94c58fa65936b8/src/elements/element.arc.js#L348

I just saw that if there are 2 arcs of 180 the error would continue to happen.